### DISCUSSION TITLE
Preview multiple changes to a space before publishing them.

### DISCUSSION THREAD

**Discussion by jalaziz** [GitbookIO/community#67] [Feature Requests]
### What's your feature/idea?

As part of our doc writing flow, we'd like to line up publications with product releases. A documentation update with a product release may contain many small changes.

Today, one way to do this today would be to have multiple spaces (a dev/staging one and a prod one), each pointed at a separate branch. The challenge with this approach is that it doesn't lend itself well to a process where there's a unique branch per releaes.

This is a more of a meta concept that could be solved in a variety of ways.

Some ideas:
1. Support multiple changes to a space that can be previewed before publishing.
2. Add API support for changing the branch that a space points to, allowing automation to update the branch on release.

### What's your desired outcome?

I would like to be able to preview multiple incremental changes to a space before publishing them.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

While we could just have multiple spaces and update the branch manually on each release, it's very manual, error-prone, and generally not a great experience (you have to reconnect to Github each time).

**Comment by taszap** [comment]
Hey @jalaziz a while ago we created a "Preview" button for change requests (available only for published spaces). This should allow you to preview all the changes you've made viewing them as a reader visiting your space would. That should address (at least partially) your request 

![image](https://github.com/GitbookIO/community/assets/63643069/e34e8a48-1fe6-4fc7-9699-0d318b9068c8)
